docker-compose.dev.yml

```
@@ -43,4 +43,5 @@ volumes:
 
 networks:
   default:
+    external: true
     name: techtrend_network
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)